k8s.gcr.io/manifests/k8s-staging-perf-tests/promoter-manifest.yaml

```diff
@@ -1,14 +1,50 @@
 # google group for gcr.io/k8s-staging-perf-tests is k8s-infra-staging-perf-tests@kubernetes.io
-
```

Comment:
This PR is slightly different from the other ones.

perf-tests was promoting to a non existent destination.

```
 REDACTED  MCW0CDP3YY  tmp  test  $  gcrane ls us.gcr.io/k8s-perf-tests
gcrane 2022/10/04 19:37:06 No matching credentials were found for "us.gcr.io"
Error: GET https://us.gcr.io/v2/token?scope=repository%3Ak8s-perf-tests%3Apull&service=us.gcr.io: UNKNOWN: Project 'project:k8s-perf-tests' not found or deleted.
 REDACTED  MCW0CDP3YY  tmp  test  ERROR  $  gcrane ls eu.gcr.io/k8s-perf-tests
2022/10/04 19:37:13 No matching credentials were found for "eu.gcr.io"
Error: GET https://eu.gcr.io/v2/token?scope=repository%3Ak8s-perf-tests%3Apull&service=eu.gcr.io: UNKNOWN: Project 'project:k8s-perf-tests' not found or deleted.
 REDACTED  MCW0CDP3YY  tmp  test  ERROR  $  gcrane ls asia.gcr.io/k8s-perf-tests
2022/10/04 19:37:19 No matching credentials were found for "asia.gcr.io"
Error: GET https://asia.gcr.io/v2/token?scope=repository%3Ak8s-perf-tests%3Apull&service=asia.gcr.io: UNKNOWN: Project 'project:k8s-perf-tests' not found or deleted.
```